docs/NextJs.md

```diff
@@ -14,6 +14,45 @@ Next.js 13 proposes 2 ways to build a React project:
 
 React-admin supports both ways. 
 
+## Prevent SSR
+React-admin is designed as a Single-Page Application, rendered on the client-side. It comes with its own routing sytem, which conflicts with the Next.js routing system. So we must prevent Next.js from rendering the react-admin component on the server-side. 
+
+To do that, we will have to create a component to bootstrap the `<Admin>` application, and import it in Next.js by using the [__lazy loading__ system provided by Next.js](https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading) and specify the [`ssr` option to false](https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading#with-no-ssr). 
+
+Using dynamic import allows disabling Server-Side Rendering for the `<Admin>` component.
+
+The following code demonstrate how we will do this further in this tutorial.
+
+Creating the `<Admin>` application as usual to bootstrap it: 
+```tsx
+// in src/App.jsx or elsewhere, depending of Next.js Router system (Pages router or App router)
+import * as React from "react";
+import { Admin, Resource, ListGuesser, EditGuesser } from 'react-admin';
+import jsonServerProvider from 'ra-data-json-server';
+
+const dataProvider = jsonServerProvider('https://jsonplaceholder.typicode.com');
+
+const App = () => (
+  <Admin dataProvider={dataProvider}>
+    <Resource name="users" list={ListGuesser} edit={EditGuesser} recordRepresentation="name" />
+    <Resource name="posts" list={ListGuesser} edit={EditGuesser} recordRepresentation="title" />
+    <Resource name="comments" list={ListGuesser} edit={EditGuesser} />
+  </Admin>
+);
+
+export default App;
+```
+Import the Admin application in Next.js:
+```tsx
```

Comment:
Would be better to add some space between the paragraphs. You may want to configure Prettier to also format your markdown files.